src/data/analyse_de.json

```diff
@@ -0,0 +1,309 @@
+            "timerange": "Zeitraum",
+            "inputs": {
+                "from": "Von",
+                "to": "bis",
+                "placeholder": "dd.mm.yyyy"
+            },
+            "cancel": "Abbrechen",
+            "apply": "Übernehmen"
+        },
+        "radio": [
+            {
+                "label": "Letzte 28 Tage",
+                "value": "1month",
+                "active": true
+            },
+            {
+                "label": "Letzte 6 Monate",
+                "value": "6months"
+            },
+            {
+                "label": "Gesamter Zeitraum",
+                "value": "all"
+            }
+        ]
+    },
+    "switch": [
+        {
+            "id": "2",
+            "label": "Täglich"
+        },
+        {
+            "id": "3",
+            "label": "Wöchentlich"
+        },
+        {
+            "id": "4",
+            "label": "7-Tage-Mittelwert"
+        },
+        {
+            "id": "1",
+            "label": "Kumuliert"
+        }
+    ],
+    "allExport": {
+        "title": "Alle Daten exportieren",
+        "link": "https://obs.eu-de.otc.t-systems.com/obs-public-dashboard/csv/cwa_all_data.zip"
+    },
+    "boards": {
+        "download": true,
+        "export": {
+            "btn": "Exportieren",
+            "data": "Daten exportieren",
+            "image": "Bild exportieren"
+        },
+        "loading": "Lade...",
+        "loadingBtn": "Erneut versuchen",
+        "total": "Gesamt (Anzahl bis <span></span>)"
+    },
+    "board": [
+        {
+            "title": "Downloads",
+            "export": "https://obs.eu-de.otc.t-systems.com/obs-public-dashboard/csv/cwa_downloads_data.zip",
+            "info": {
+                "content": "<strong>Täglich</strong><p>Tagesgenaue Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores seit Veröffentlichung der Corona-Warn-App.</p><br><strong>Weitere Informationen</strong><p>In die Erhebung der Downloadzahlen gehen nur die in der Corona-Warn-App-Downloads aus den offiziellen App-Stores von Google und Apple ein.</p><p>Aus diesen Zahlen können keine Rückschlüsse bezüglich der aktiven Nutzerbasis gezogen werden. </p>"
```

Comment:
The sentence "In die Erhebung der Downloadzahlen gehen nur die in der Corona-Warn-App-Downloads aus den offiziellen App-Stores von Google und Apple ein." doesn't really make sense to me, or am I just not understanding it?

Better would be: 
```suggestion
                "content": "<strong>Täglich</strong><p>Tagesgenaue Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der Corona-Warn-App-Downloads aus Google Play (Google) und dem App Store (Apple) sowie die Summe der Corona-Warn-App-Downloads aus beiden App-Stores seit Veröffentlichung der Corona-Warn-App.</p><br><strong>Weitere Informationen</strong><p>In die Erhebung der Downloadzahlen gehen nur die Downloads der Corona-Warn-App aus den offiziellen App-Stores von Google und Apple ein.</p><p>Aus diesen Zahlen können keine Rückschlüsse bezüglich der aktiven Nutzerbasis gezogen werden. </p>"
```